src/scripting/lua_map.cc

```diff
@@ -5874,6 +5900,25 @@ int LuaShip::get_shipname(lua_State* L) {
 	return 1;
 }
 
+/* RST
+   .. attribute:: capacity
+
+   The ship's current capacity. Defaults to the capacity defined in the tribe's singleton ship description.
+
+   Do not change this value if the ship is currently shipping more items than the new capacity allows.
```

Comment:
Add some error handling for this case, so that the player/scenario editor will get a good error message rather than some undefined crash boom bang.